tests/integration/test_statistics.py

```diff
@@ -0,0 +1,51 @@
+import dask.dataframe as dd
+import pandas as pd
+import pytest
+
+from dask_sql.physical.utils.statistics import parquet_statistics
+
+
+@pytest.mark.parametrize("parallel", [None, False, 2])
+def test_parquet_statistics(parquet_ddf, parallel):
```

Comment:
Since these checks are against a (presumably) internal util would it make sense to move this file to `tests/unit/`?

Reply:
yes - that makes sense to me!